### PR TITLE
[IMP] fields: dummy updates of one2many fields should not trigger recomputations

### PR DESCRIPTION
**Pull request by rco-odoo**

### CODE REVIEW

**Comment by rco-odoo** [comment]
@sle-odoo this should fix the issue :smile:

**Comment by rco-odoo** [comment]
@robodoo r+

**Comment by rco-odoo** [comment]
@robodoo r-

File: addons/stock/models/stock_picking.py [hunk 1]
```diff
@@ -637,7 +637,7 @@ def action_done(self):
         self.write({'date_done': fields.Datetime.now()})
         return True
 
-    @api.depends('state', 'move_lines', 'move_lines.state', 'move_lines.package_level_id')
```
Comment (Whenrow) [inline addons/stock/models/stock_picking.py]: https://xkcd.com/1172/

**Comment by rco-odoo** [comment]
@robodoo rebase-ff r+

**Pull request closed without merging**